datajunction-server/datajunction_server/api/helpers.py

```diff
@@ -817,6 +818,7 @@ def build_sql_for_multiple_metrics(  # pylint: disable=too-many-arguments,too-ma
         session,
         metrics,
         dimensions,
+        check_dimensions=False,
```

Comment:
Why we don't need to "check dimensions" in a cube?

Reply:
@shangyian can correct me but it looks like this `check_dimensions` flag just checks that dimensions have been provided which is something we already check at the front of the API call logic. The logic here still validates the provided dimension attributes exist and are compatible with the metrics, regardless of the value for the `check_dimensions` arg. Maybe that arg could have a better name? Something like `dimensions_required`? Or maybe we should leave this check out completely from the `validate_cube` inner logic and leave it to the calling code to do the `if not dimensions:` check and raise if it needs to.

Reply:
Yeah, I agree that it should be renamed -- I changed it to `require_dimensions`. I opted not to remove it because it's called in five other places, and it's nicer not to repeat the logic that many times.